### PR TITLE
MNT: Re-rendered with conda-smithy 3.1.12 and pinning 2018.11.27

### PR DESCRIPTION
**Pull request by leouieda**
Needed for conda-forge/gmt-feedstock#57

### CODE REVIEW

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender

File: conda-forge.yml [hunk 1]
```diff
@@ -4,3 +4,6 @@ travis:
 appveyor:
   secure:
     BINSTAR_TOKEN: tumuXLL8PU75WMnRDemRy02ruEq2RpNxeK3dz0MjFssnosPm2v4EFjfNB4PTotA1
+compiler_stack: comp7
```
Comment (ocefpaf) [inline conda-forge.yml]: We need this to upload the package to the `gcc7` channel. There is no compilation part here but the recipe is "arch" in a way due to the patch differences in the platforms.

I'm not sure why the migrator did not catch these recipes. Maybe @CJ-Wright may help here.

PS: another alternative would be to "copy" these to the `gcc7` label without a rebuild.

Comment (leouieda) [reply conda-forge.yml]: Thanks, Filipe. I'll make the same change to DCW as well and bumb the build number. Should be quick to rebuild these.

Comment (CJ-Wright) [reply conda-forge.yml]: I'm not certain why the migrator didn't PR this at the beginning, maybe because it doesn't have requirements?

**Pull request merged**